src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx

```diff
@@ -71,38 +73,41 @@ export class VanguardArtistWrapper extends React.Component<
     const url = `url("data:image/svg+xml,${svgString}")`
 
     return (
-      <SVGBackground url={url}>
-        <ArtistContainer pb={4} maxWidth={1000} px={4} mx="auto">
-          <Box textAlign="center" pt={50}>
-            <ArtistTitle size="8">{title}</ArtistTitle>
-            <Box position="absolute">
-              <Share
-                // TODO: We may need to use custom urls for in-page routing
-                url={getFullEditorialHref(layout, slug)}
-                title={title}
-              />
-            </Box>
-            {hero_section && (
-              <Sans size="4" weight="medium">
-                {hero_section.deck}
-              </Sans>
-            )}
-          </Box>
+      <FullScreenProvider>
+        <SVGBackground url={url}>
+          <ArticleWithFullScreen article={article}>
+            <ArtistContainer pb={4} maxWidth={1000} px={4} mx="auto">
+              <Box textAlign="center">
+                <ArtistTitle size="8">{title}</ArtistTitle>
+                <Box position="absolute">
+                  <Share
+                    // TODO: We may need to use custom urls for in-page routing
+                    url={getFullEditorialHref(layout, slug)}
+                    title={title}
+                  />
+                </Box>
+                {hero_section && (
+                  <Sans size="4" weight="medium">
+                    {hero_section.deck}
+                  </Sans>
+                )}
+              </Box>
+              {/** TODO: Sections may need to be customized to handle expansion */}
+              <Sections hideAds article={article} customWidth={900} />
 
-          {/** TODO: Sections may need to be customized to handle expansion */}
-          <Sections hideAds article={article} />
-
-          <ReadMoreButton
-            size="5"
-            weight="medium"
-            textAlign="center"
-            onClick={this.onExpand}
-          >
-            {isExpanded ? "Read More" : "Close"}
-            <Sans size="8">{isExpanded ? "\u2193" : "\u2191"}</Sans>
-          </ReadMoreButton>
-        </ArtistContainer>
-      </SVGBackground>
+              <ReadMoreButton
+                size="5"
+                weight="medium"
+                textAlign="center"
+                onClick={this.onExpand}
+              >
+                {isExpanded ? "Read More" : "Close"}
+                <Sans size="8">{isExpanded ? "\u2193" : "\u2191"}</Sans>
```

Comment:
As previously mentioned, these unicode items are kinda vague; why not just grab a chevron arrow from palette?